src/CMakeLists.txt

```
@@ -1,3 +1,15 @@
+add_library(mc_state_observation SHARED conversions/kinematics.cpp
+                                        odometry/LeggedOdometryManager.cpp)
```

```suggestion
set(mc_state_observation_SRC conversions/kinematics.cpp odometry/LeggedOdometryManager.cpp)
set(mc_state_observation_HDR ...)
add_library(mc_state_observation SHARED ${mc_state_observation_SRC} ${mc_state_observation_HDR})
```

The headers are not required but they provide a nicer experience in IDE projects